src/Miningcore/Payments/PayoutHandlerBase.cs

```diff
@@ -179,7 +179,7 @@ protected virtual async Task PersistPaymentsAsync(Balance[] balances, string tra
         public string FormatAmount(decimal amount)
         {
             var coin = poolConfig.Template.As<CoinTemplate>();
-            return $"{amount:0.#######} {coin.Symbol}";
+            return $"{amount:0.#########} {coin.Symbol}";
```

Comment:
Got this change covered in my PR too. Whoever goes first following one should merge the fix.